mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -0,0 +1,77 @@
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import shutil
+
+import warnings
+from nose.tools import assert_raises, assert_equal
+from numpy.testing import assert_array_equal
+
+from mne import write_events, read_epochs_eeglab
+from mne.io import read_raw_eeglab
+from mne.io.tests.test_raw import _test_raw_reader
+from mne.datasets import testing
+from mne.utils import _TempDir, run_tests_if_main, requires_version
+
+base_dir = op.join(testing.data_path(download=False), 'EEGLAB')
+raw_fname = op.join(base_dir, 'test_raw.set')
+raw_fname_onefile = op.join(base_dir, 'test_raw_onefile.set')
+epochs_fname = op.join(base_dir, 'test_epochs.set')
+epochs_fname_onefile = op.join(base_dir, 'test_epochs_onefile.set')
+montage = op.join(base_dir, 'test_chans.locs')
+
+warnings.simplefilter('always')  # enable b/c these tests throw warnings
+
+
+@requires_version('scipy', '0.12')
+@testing.requires_testing_data
+def test_io_set():
+    """Test importing EEGLAB .set files"""
+    from scipy import io
+
+    _test_raw_reader(read_raw_eeglab, input_fname=raw_fname, montage=montage)
+    with warnings.catch_warnings(record=True) as w:
+        _test_raw_reader(read_raw_eeglab, input_fname=raw_fname_onefile,
+                         montage=montage)
+    assert_equal(len(w), 2)  # preload=False or str throw warnings
+
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter('always')
+        epochs = read_epochs_eeglab(epochs_fname)
+        epochs2 = read_epochs_eeglab(epochs_fname_onefile)
+    assert_equal(len(w), 6)  # 3 warnings for each read_epochs_eeglab
```

Comment:
That's alright. Normally users will only see one (assuming the message is identical) but we get 3 because of `simplefilter('always')`. It should be fine just to add a comment that there are 3 epochs that are assoc with multiple events.